doc/api/inspector.md

```diff
@@ -108,6 +108,49 @@ with an error. [`session.connect()`] will need to be called to be able to send
 messages again. Reconnected session will lose all inspector state, such as
 enabled agents or configured breakpoints.
 
+## inspector.ws
+
+The Inspector can be interacted with via a websocket-based protocol. These APIs
+allow programatic control of the websocket, equivalent to use `--inspect*` CLI
+options.
+
+### ws.listen()
```

Comment:
Maybe ws.listen(waitForFrontend[, host][, port])?

Also, I feel like the agent should always be started from JavaScript - node_bootstrap.js or something can call ws.listen based on command line arguments.

I was planning exposing something like `inspector.onDebugSignal(callback)` that would allow developers to customize WS server startup - e.g. to start a server on a different port.